spec/board_spec.rb

```diff
@@ -0,0 +1,66 @@
+require './lib/ship'
+require './lib/cell'
+
+RSpec.describe Board do
+
+  before(:each) do
+    @board = Board.new
+  end
+
+  describe 'initialize' do
```

Comment:
Use # in front of method names for describe blocks. '#initialize'

Reply:
Updated